web/_js/pointInPolygon.js

```diff
@@ -45,3 +45,6 @@ function pointIsInPolygon (point, polygon) {
     
     return inside;
 };
+
+if( typeof window == "undefined") // for use in /tools using nodejs
+    module.exports = pointIsInPolygon;
```

Comment:
```suggestion

// for use in node.js tooling
if (typeof window == "undefined") {
    module.exports = pointIsInPolygon;
}
```

Include a trailing newline to be [POSIX compliant](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).